README.md

```diff
@@ -65,35 +65,20 @@ structures and functions. You are not that limited with open-source engine.
 OpenTomb is a cross-platform engine — currently, you can run it on Windows, Mac or Linux.
 No mobile implementations are made yet, but they are fully possible.
 
-### Configuration and autoexec files ###
-Currently, all settings in OpenTomb are managed through configuration and autoexec files.
-Configuration file contains persistent engine and game settings, while autoexec contains
-any commands which should be executed on engine start-up.
-
-Configuration file (config.lua) is divided into different sections: screen, audio, render,
-controls, console and system. In each of these sections, you can change numerous parameters,
-which names are usually intuitive to understand.  
-Autoexec file (autoexec.lua) is a simple command file which is executed at engine start-up,
-just like you type them in the console. Basically, you shouldn't remove any existing commands
-from autoexec, as most likely engine won't start properly then, but you can modify these
-commands or add new ones — like changing start-up level by modifying setgamef() command.
-
-### Installation and running ###
+### Setup ###
 You don't need to install OpenTomb, but you need all classic TR game resources. Problem is,
-these resources (except level files themselves) are tend to be in some cryptic formats or
-incompatible across game versions. Because of this, you need to convert some game resources
-by yourself or get them from somewhere on the Net. Anyway, here is the list of all needed
-assets and where to get them:
+these resources (except level files themselves) tend to be in some cryptic formats, or
+incompatible across game versions. Because of this, you'll need to convert some game resources yourself,
+or get them from somewhere on the Net. Anyway, here is the list of all needed assets and where to get them:
 
  * Data folders from each game. Get them from your retail game CDs or Steam/GOG bundles.
    Just take data folder from each game's folder, and put it into corresponding
-   /data/tr*/ folder.
+   /data/tr*/ folder. For instance, for TR3, the path would be OpenTomb/data/tr3/data/
```

Comment:
The exact pathing of the data folders is a little unclear at first. Hopefully an example clears things up.